xlive/H2MOD.cpp

```diff
@@ -39,6 +39,7 @@ bool b_XboxTick = false;
 bool b_Infection = false;
 bool b_Halo2Final = false;
 bool b_HeadHunter = false;
+bool b_inGame = false;
```

Comment:
don't set booleans for whether you are in game or not, actually find the h2v engines way of determining if you are in-game